server/services/bookingService.ts

```diff
@@ -82,6 +82,34 @@ export default class BookingService {
     ])
   }
 
+  async currentResidents(premisesId: string): Promise<Array<TableRow>> {
+    const token = 'FAKE_TOKEN'
+    const bookingClient = this.bookingClientFactory(token)
+
+    const bookings = await bookingClient.allBookingsForPremisesId(premisesId)
+
+    return this.currentResidentsToTableRows(bookings, premisesId)
```

Comment:
This doesn't seem to filter for residents who have arrived?

Reply:
oops!